core/src/software/aws/toolkits/core/credentials/ToolkitCredentialsProvider.kt

```diff
@@ -6,6 +6,12 @@ package software.aws.toolkits.core.credentials
 import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider
 
 interface CredentialIdentifier {
+    /**
+     * The ID must be unique across all [CredentialIdentifier].
```

Comment:
feels like this sentence is missing "instances" at the end.

Reply:
Fixed